tensorflow/core/distributed_runtime/rpc/grpc_server_lib.h

```diff
@@ -36,6 +36,13 @@ namespace tensorflow {
 class GrpcWorker;
 class Master;
 
+typedef std::function<RendezvousMgrInterface*(const WorkerEnv*, 
+          const std::string& worker_name, WorkerCacheInterface* worker_cache)> 
+        RendezvousMgrCreationFunction;
+
+typedef std::function<void(const WorkerEnv*, ::grpc::ServerBuilder*)> 
```

Comment:
"ServiceCreationFunction" registers a service on the server. This is one way to register a service by the derived class. VerbsServer::Init() calls GrpcServer::Init(). But the server is already built and run after GrpcServer::Init(), so I have to pass a function to GrpcServer::Init() to register an extra service.